inst/include/common/fims_math.hpp

```diff
@@ -86,6 +86,24 @@ inline const T logistic(const T &median, const T &slope, const T &x) {
   return (1.0) / (1.0 + exp(-1.0 * slope * (x - median)));
 }
 
+/**
+ * @brief The general double logistic function
+ *
+ * \f$ \frac{1.0}{ 1.0 + exp(-1.0 * slope_{1} (x - median_{1}))} \left(1-\frac{1.0}{ 1.0 + exp(-1.0 * slope_{2} (x - median_{2}))}  \right)\f$
+ *
+ * @param median1 the median (inflection point) of the ascending limb of the double double logistic function
+ * @param slope1 the slope of the aascending limb of e double hlogistic function
```

Comment:
extra "a" in ascending

Reply:
Issue was discussed and subgroup had strong preference but noted that
(i) if you use 1 and 2 then there is an implied ordering of smaller to larger
for the inflection point values, so median1 < median2 or median_asc < median_des
(ii) if one thought of adding general logistic function at some point with more
than 2 limbs then using a numbers would be a simpler labeling approach as in point (i)
(iii) it seems like adding a comment would help, something like, 
"input median1 < median2"